app/src/main/java/com/track/finance2up/ui/theme/Theme.kt

```diff
@@ -49,7 +49,7 @@ fun Finance2UpTheme(
             val context = LocalContext.current
             if (darkTheme) dynamicDarkColorScheme(context) else dynamicLightColorScheme(context)
         }
-        darkTheme -> DarkColorScheme
+//        darkTheme -> DarkColorScheme
```

Comment:
e đang muốn disable dark theme nhỉ? Nếu vậy thì đoạn này a nghĩ đổi cái default arg ở dòng 42 thành false thì ngon hơn, về sau mình cần thì chỉ cần thay đổi ở đó. Hiện tại nếu disable phần này thì với bản Android có Material You vẫn vào đc dark theme (dòng 50)